commit 55f957b41fe639089184397fdf00d6b4d5fcfe69 (refs/changes/84/2284/2)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2017-07-31T11:32:30-07:00 (2 years, 2 months ago)
    
    TOOLS-1823 split sdcadm experimental volapi into sdcadm postsetup and sdcadm experimental commands

diff --git a/tools/setup/setup.sh b/tools/setup/setup.sh
index f5fc4bd..b3e19ba 100644
--- a/tools/setup/setup.sh
+++ b/tools/setup/setup.sh
@@ -275,8 +275,8 @@ else
     fatal "Could not find latest sdcadm version with tritonnfs support"
 fi
 
-echo "Enabling experimental VOLAPI service"
-sdcadm experimental volapi
+echo "Setting up VOLAPI service"
+sdcadm post-setup volapi
 
 upgrade_core_service_to_latest_branch_image "sdc" "tritonnfs"
 upgrade_core_service_to_latest_branch_image "workflow" "tritonnfs"
@@ -291,10 +291,8 @@ upgrade_core_service_to_latest_branch_image "volapi" "tritonnfs"
 # Needed to add additional programs in the HN's GZ, such as sdc-volapi
 upgrade_gz_tools_to_latest_branch_image tritonnfs
 
-echo "Restarting sdc-docker to account for configuration changes..."
-/opt/smartdc/bin/sdc-login -l docker svcadm restart config-agent
-/opt/smartdc/bin/sdc-login -l docker svcadm restart docker
-echo "sdc-docker restarted!"
+# Now enable the experimental_nfs_shared_volumes flag in SAPI
+sdcadm experimental nfs-volumes
 
 EOS
 
